apps/nextjs/src/stores/chatStore/types.ts

```diff
@@ -58,6 +59,7 @@ export type ChatState = {
     append: (action: ChatAction) => void;
     stop: () => void;
     streamingFinished: () => void;
+    streamingFailed: () => void;
```

Comment:
Very minor (and just personal preference) but I like it when naming of a callback like this signifies that it's an action (e.g. onStreamingFailed, or a verb like 'stop..' or 'append..' etc), to help distinguish it from state properties when scanning.